course6/toyapp-adam/math/matrix.cc

```diff
@@ -6,14 +6,20 @@ namespace Math {
 
 namespace {
 	constexpr auto PI = std::atan(1.0) * 4.0;
+	constexpr auto TRANSLATE_X = 3;
```

Comment:
constexpr doesn't mean anything in this case. Just use ```const```

Reply:
But it does for PI, right?